TheWarWithin/DemonHunterVengeance.lua

```diff
@@ -1203,14 +1228,14 @@ spec:RegisterAbilities( {
         talent = "demonic_intensity",
         startsCombat = true,
         texture = 135798,
-        buff = "metamorphosis",
+        buff = "demonsurge_hardcast",
 
         start = function ()
```

Comment:
Some of this function can be simplified by calling `spec.abilities.fel_devastation.start()` instead of duplicating the logic.

Similarly for the `finish` function which can just be the one-liner call to `spec.abilities.fel_devastation.finish()`